lib/facilities/vha_facility.rb

```diff
@@ -6,36 +6,62 @@ class << self
       attr_writer :validate_on_load
 
       def pull_source_data
-        metadata = Facilities::MetadataClient.new.get_metadata(arcgis_type)
+        get_all_the_facilities_data.map(&method(:new))
+      end
+
+      def get_all_the_facilities_data
+        gis_type = 'FacilitySitePoint_VHA'
+        sort_field = 'Sta_No'
+        metadata = Facilities::GisMetadataClient.new.get_metadata(gis_type)
         max_record_count = metadata['maxRecordCount']
-        Facilities::Client.new.get_all_facilities(arcgis_type, sort_field, max_record_count).map(&method(:new))
+        resp = Facilities::GisClient.new.get_all_facilities(gis_type, sort_field, max_record_count)
```

Comment:
Are other classes going to use the `GisClient` and `GisMetadataClient`?  
If not, you can hard code `gis_type` and `sort_field` in the `GisClient` and `GisMetadataClient` classes instead of passing them as parrameters.

Reply:
Yes, probably.

Reply:
Yes, in the near future we will begin using `GisClient` to pull Vet Centers.